src/core/Canvas.js

```diff
@@ -15,6 +15,25 @@ export class Canvas extends BaseCanvas {
         this._defs = this.createDefs();
     }
 
+    getElementRegistry () {
+        return this._elementRegistry;
```

Comment:
:grin:

Reply:
You noticed that we had to access it with a underscore, which normally means that that is a private attribute, so I created a getter. :D